keyboards/splitkb/aurora/sweep/rev1/config.h

```diff
@@ -29,7 +29,7 @@
 // Not yet available in `info.json`
 #ifdef RGB_MATRIX_ENABLE
 #    define RGB_MATRIX_MAXIMUM_BRIGHTNESS RGBLIGHT_LIMIT_VAL
-#    define DRIVER_LED_TOTAL RGBLED_NUM
+#    define RGB_MATRIX_LED_COUNT RGBLED_NUM
```

Comment:
This should probably be reverted.

Reply:
Whoops! That definitely shouldn't have gone in here!